app/controllers/dragons_teams_controller.rb

```diff
@@ -0,0 +1,51 @@
+class DragonsTeamsController < ApplicationController
+  def index
+    @view_model = UserHomePageViewModel.new
+  end
+
+  def show
+    @view_model = UserHomePageViewModel.new
+    @user = current_user
+    @dragon = @user.dragons.find(params[:id])
+  end
+
+  def new
+    @view_model = UserHomePageViewModel.new
+    @user = @view_model.current_user
+    @dragon = Dragon.new
+    # @dragon = current_user.dragon.build
+  end
+
+  def create
+    @view_model = UserHomePageViewModel.new
+    @user = @view_model.current_user
+    if @user.dragons.count >= 5
+      render 'index'
+    else
+      add_dragon
+    end
+  end
+
+  def destroy
+    @view_model = UserHomePageViewModel.new
```

Comment:
`@view_model = UserHomePageViewModel.new`
czy to jest w ogóle potrzebne tutaj? nie wyświetlamy żadnych danych w akcji `destroy`